crates/settlement-clients/ethereum/src/lib.rs

```diff
@@ -27,10 +27,10 @@ use mockall::{automock, predicate::*};
 
 use alloy::providers::ProviderBuilder;
 use conversion::{get_input_data_for_eip_4844, prepare_sidecar};
-#[cfg(feature = "testing")]
-use settlement_client_interface::{SettlementClient, SettlementConfig, SettlementVerificationStatus};
 #[cfg(not(feature = "testing"))]
-use settlement_client_interface::{SettlementClient, SettlementConfig, SettlementVerificationStatus};
+use settlement_client_interface::{SettlementClient, SettlementVerificationStatus};
+#[cfg(feature = "testing")]
+use settlement_client_interface::{SettlementClient, SettlementVerificationStatus};
```

Comment:
what's the different in testing and not testing?

Reply:
yes this was a mistake, removed extra code.